READMEs/marketplace-flow.md

```diff
@@ -64,6 +64,13 @@ Then, please refer to [publish-flow](publish-flow.md) and complete the following
 - [x] 2. Publish Dataset
 
 
+The asset metadata stored on-chain is encrypted and compressed by default.
```

Comment:
the previous lines pertaining to this explanation were moved to the publish flow. I like the more detailed explanation, but please move it to the publish flow and remove from here.

Reply:
Sure I can move this to the publish flow.

Reply:
Done.